src/templates/app/cashier/index.jsx

```diff
@@ -139,6 +139,30 @@ const Cashier = () => (
                 <DepositWithdrawButton is_payment_agent />
             </div>
         </div>
+
+        <div className='gr-padding-10' />
+
+        <div className='gr-padding-10 table-body invisible gr-parent' id='dp2p_info'>
+            <h3 className='gr-padding-10'>{it.L('DP2P')}</h3>
```

Comment:
are we calling it DP2P in binary? let's discuss this

Reply:
have we discuss this yet?

Reply:
yes, please change it to P2P